scripts/jenkins/qa_openstack.sh

```diff
@@ -167,9 +167,10 @@ set -e
 $zypper -n install -t pattern cloud_controller cloud_compute cloud_network
 $zypper -n install --force openstack-quickstart openstack-tempest-test
 
-# test -e /tmp/openstack-quickstart-demosetup && mv /tmp/openstack-quickstart-demosetup /usr/sbin/openstack-quickstart-demosetup
-# test -e /tmp/keystone_data.sh && mv /tmp/keystone_data.sh /usr/lib/devstack/keystone_data.sh
-# test -e /tmp/functions.sh && mv /tmp/functions.sh /usr/lib/openstack-quickstart/functions.sh
+test -e /tmp/openstack-quickstart-demosetup && cp /tmp/openstack-quickstart-demosetup /usr/sbin/openstack-quickstart-demosetup && echo "WARN: using /tmp/openstack-quickstart-demosetup for openstack-quickstart"
```

Comment:
Please can you not use incredibly long lines?  Just because the `E006` `bashate` test is disabled for Travis, it doesn't mean it's OK to add new violations of it ;-)